libyul/optimiser/Suite.cpp

```diff
@@ -472,7 +472,7 @@ void OptimiserSuite::runSequence(std::string_view _stepAbbreviations, Block& _as
 			subsequences.push_back({subsequence, true});
 	}
 
-	size_t codeSize = 0;
+	size_t codeSize = (_repeatUntilStable ? CodeSize::codeSizeIncludingFunctions(_ast) : 0);
```

Comment:
The ternary seems redundant, no? If `_repeatUntilStable` is `false`, it'll break the loop, and will thus not reach the code size check?

Reply:
Forgot to get back to this one. Now that I look at the code again, I remembered that it was actually on purpose. The value is irrelevant but the point is that we don't have to run the metric to get it, which is a complex and potentially heavy operation so it's a waste to run it more often than necessary.

I added a comment to make it more obvious.